tests/test_database/charleston/input/events/ETA/ETA.toml

```diff
@@ -15,7 +15,7 @@ source = "none"
 [rainfall]
```

Comment:
We grab the test_database from the P drive right? Then why are we making changes to this (or why is it here at all)

Reply:
The test_database is last edited 4 months ago so no way it is up-to-date